src/ssl.c

```diff
@@ -19317,7 +19317,6 @@ void wolfSSL_ACCESS_DESCRIPTION_free(WOLFSSL_ACCESS_DESCRIPTION* access)
         wolfSSL_ASN1_OBJECT_free(access->method);
     if (access->location)
         wolfSSL_GENERAL_NAME_free(access->location);
-    access = NULL;
```

Comment:
In these cases is better to keep the code to set the pointer to NULL to make sure that any future code does not double free or try accessing an address that has already been free'd